extensions/host-switch/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Localhosty Changelog
```

Comment:
Localhosty? ❤️

Reply:
ah, that was initial name, forgot to cleanup 😄 
fixed now